torchci/rockset/utilization/__sql/runner_utilization_by_repo.sql

```diff
@@ -0,0 +1,25 @@
+SELECT
+  FORMAT_ISO8601(
+      DATE_TRUNC(
+          :granularity,
+          started_at AT TIME ZONE :timezone
+      )
+  ) AS started_date,
+  project, 
+    SUM(DATE_DIFF('SECOND',started_at ,completed_at)) AS duration,
+FROM (SELECT
+  PARSE_TIMESTAMP_ISO8601(started_at) as started_at,
+  PARSE_TIMESTAMP_ISO8601(completed_at) as completed_at,
+  ELEMENT_AT(SPLIT(SUBSTR(run_url, 30, 50), '/'), 2) AS project,
```

Comment:
Is the length `50` a required param?  I think we can remove it if it's optional

Reply:
Length is optional, removing.